scripts/generate_ssl_debug_helpers.py

```diff
@@ -287,7 +287,7 @@ def __str__(self):
             {translation_table}
                 }};
 
-                return "UNKOWN";
+                return "UNKNOWN";
```

Comment:
this isn't a typo in a comment, it's in a string value which could potentially be used in other places. are we sure this change has no impact?

Reply:
@yuhaoth introduced this script so may be able to comment. It seems safe - the code is a helper function used for debug so should not be parsed elsewhere.

Reply:
Okay, yes, looks like this outputs strings that are used in debug statements, so should be fine.

Related, I notice at line 227 in this file it also has

`return "UNKOWN_VAULE";`

which causes 4 of those to appear in `library/ssl_debug_helpers_generated.c`.

And there's also a typo at the top of this file (`scripts/generate_ssl_debug_helpers.py`) where it says

`"""Generate library/ssl_debug_helps_generated.c`

(it actually creates `library/ssl_debug_helpers_generated.c`)

Reply:
These are addressed separately in https://github.com/Mbed-TLS/mbedtls/pull/5773